### ISSUE TITLE
Unable to subscribe as an existing user

### ISSUE THREAD

**Issue by gabe-connolly** [zeen101/leaky-paywall#88]
When I try to subscribe as an existing user, I get the following error:

`
Notice: Undefined index: password in /var/www/html/wp-content/plugins/leaky-paywall/functions.php on line 2451 Warning: Cannot modify header information - headers already sent by (output started at /var/www/html/wp-content/plugins/leaky-paywall/functions.php:2451) in /var/www/html/wp-includes/pluggable.php on line 932 Warning: Cannot modify header information - headers already sent by (output started at /var/www/html/wp-content/plugins/leaky-paywall/functions.php:2451) in /var/www/html/wp-includes/pluggable.php on line 933 Warning: Cannot modify header information - headers already sent by (output started at /var/www/html/wp-content/plugins/leaky-paywall/functions.php:2451) in /var/www/html/wp-includes/pluggable.php on line 934 Warning: Cannot modify header information - headers already sent by (output started at /var/www/html/wp-content/plugins/leaky-paywall/functions.php:2451) in /var/www/html/wp-includes/pluggable.php on line 1265 Warning: Cannot modify header information - headers already sent by (output started at /var/www/html/wp-content/plugins/leaky-paywall/functions.php:2451) in /var/www/html/wp-includes/pluggable.php on line 1268
`

Inspecting the `$args` array that is passed to `leaky_paywall_email_subscription_status`, I can confirm that there is no key of 'password':
```
Array
(
    [id] => 1
    [login] => username
    [email] => user@email
    [first_name] => User
    [last_name] => Name
    [need_new] => 
    [level_id] => 0
    [subscriber_id] => 
    [subscriber_email] => user@email
    [existing_customer] => 
    [price] => 0
    [description] => Magazine Subscription
    [payment_gateway] => free_registration
    [payment_status] => active
    [interval] => month
    [interval_count] => 1
    [site] => 
    [plan] => 
    [recurring] => 
    [user_id] => 1
    [transaction_id] => 267
)
```

**Expected behavior**
I would expect that even as an already registered user, I would be able to change my subscription, or failing that, have graceful error handling to avoid seeing a broken page.

**My environment**
* WordPress 5.3.1 
* Leaky Paywall 4.13.9

**Comment by greenhornet79** [comment]
Thanks for letting us know. I'll investigate and see if I can replicate the issue.

**Comment by greenhornet79** [comment]
This has been fixed in version 4.14.5
